src/InteractiveMars/Markers.ts

```diff
@@ -1,36 +1,29 @@
+import {Marker} from "react-globe";
 
-// interface Marker {
-//     id: number;
-//     rover: string;
-//     color: string;
-//     coordinates: number[];
-//     value: number;
-// }
```

Comment:
We don't need this - because ReactGlobe actually defines this itself, so we can just use that.